release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -152,9 +152,145 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Alpha3](https://githu
 
 #### Standard Library
 
-##### Bug Fixes
+##### Log Package Updates
 
-To view bug fixes, see the [GitHub milestone for Swan Lake Alpha3](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Swan+Lake+Alpha3%22).
+###### Introduce additional log levels and log functions
+
+There are 4 log levels: `DEBUG`, `ERROR`, `INFO`,and `WARN` and their respective functions: `printDebug`, `printError`, `printInfo`, and `printWarn`. To set the global log level, place the entry given below in the `Config.toml` file:
+```
+[log] 
+level = "[LOG_LEVEL]"
+```
+
+Each module can also be assigned its own log level. To assign a log level to a module, provide the following entry in the `Config.toml` file:
+```
+[[log.modules]] 
+name = "[MODULE_NAME]" 
+level = "[LOG_LEVEL]"
+```
+
+#### OS Package Updates
+
+- Removed the `exec` function.
+
+#### Task Package Updates
+
+The module has been revamped by removing the `Scheduler` and `Listener` classes and introducing the following functions to schedule and manage the job either one-time or periodically.
+
+- Configure the scheduler worker pool with the worker count and max waiting time.
+```ballerina
+task:Error? output = task:configureWorkerPool(6, 7000);
+```
+
+- Schedule the Ballerina job at a specified time.
+```ballerina
+class MyJob {
+   *task:Job;
+
+   public function execute() {
+       // logic goes here
+   }
+}
+
+ZoneOffset zoneOffset = {hours: 5, minutes: 30};
+Civil time = {
```

Comment:
Do these come from the time package?

Reply:
Yes, fixed in here https://github.com/ballerina-platform/ballerina-release/pull/114#discussion_r595718951